sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -764,13 +765,13 @@ case class DescribeColumnCommand(
     val colName = UnresolvedAttribute(colNameParts).name
     val field = {
       relation.resolve(colNameParts, resolver).getOrElse {
-        throw new AnalysisException(s"Column $colName does not exist")
+        throw QueryCompilationErrors.columnDoesNotExistError(colName)
       }
     }
     if (!field.isInstanceOf[Attribute]) {
       // If the field is not an attribute after `resolve`, then it's a nested field.
-      throw new AnalysisException(
-        s"DESC TABLE COLUMN command does not support nested data types: $colName")
+      throw QueryCompilationErrors.commandNotSupportNestedColumnError(
+        "DESC TABLE COLUMN", toPrettySQL(field, removeAlias = true))
```

Comment:
We won't hit this branch anymore, as we fail earlier in `ResolveSessionCatalog`. We can simply use `assert(field.isInstanceOf[Attribute])`

Reply:
I think this can still be hit for views.